### PR TITLE
Unnecessary "return", "continue" statements

### PR DESCRIPTION
**Pull request by YaPravaATyNet**

### CODE REVIEW

File: src/com/github/kornilova203/matlab/inspections/MatlabUnnecessaryControlExprVisitor.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import com.intellij.psi.tree.IElementType
+import com.intellij.psi.util.elementType
+
+open class MatlabUnnecessaryControlExprVisitor(private val holder: ProblemsHolder,
+                                               val type: IElementType,
+                                               val exprName: String) : MatlabVisitor() {
+    fun findUnnecessaryExpr(scope: PsiElement?) {
+        if (scope == null) return
+        var lastChild = scope.lastChild
+        while (lastChild != null && lastChild.elementType in setOf(TokenType.WHITE_SPACE, MatlabTypes.NEWLINE, MatlabTypes.SEMICOLON, MatlabTypes.FUNCTION_DECLARATION)) {
```
Comment (kornilova203) [inline src/com/github/kornilova203/matlab/inspections/MatlabUnnecessaryControlExprVisitor.kt:19]: `PsiTreeUtil.skipSiblingsBackward`

**Review by kornilova203 — CHANGES_REQUESTED** [review]
Wrong formatting after applying a quick fix:
```matlab
% before
while true
  if a
    continue
  end 
end

% after
while true
  if a
      end
end
```

File: src/com/github/kornilova203/matlab/inspections/MatlabUnnecessaryControlExprVisitor.kt (hunk 1 above)
Comment (YaPravaATyNet) [reply src/com/github/kornilova203/matlab/inspections/MatlabUnnecessaryControlExprVisitor.kt:19]: The arguments in PsiTreeUtil.skipSiblingsBackward must be classes. Is there some classes for tokens like semicolon?

Comment (kornilova203) [reply src/com/github/kornilova203/matlab/inspections/MatlabUnnecessaryControlExprVisitor.kt:19]: Semicolon is just a `PsiElement` so seems like `skipSiblingsBackward` doesn't work in this case

**Pull request merged**